pkg/agent/datamodel/sig_config.go

```diff
@@ -477,6 +482,20 @@ var (
 		Version:       LinuxSIGImageVersion,
 	}
 
+	SIGUbuntuFipsContainerd2204ImageConfigTemplate = SigImageConfigTemplate{
+		ResourceGroup: AKSUbuntuResourceGroup,
+		Gallery:       AKSUbuntuGalleryName,
+		Definition:    "2204fipscontainerd",
+		Version:       "202404.09.0", // TODO(artunduman): Update version when the image is ready
```

Comment:
Points to the image version being released [here](https://msazure.visualstudio.com/CloudNativeCompute/_releaseProgress?_a=release-pipeline-progress&releaseId=159123)

Reply:
we have yet to add RP selection logic for these new SKUs right?

Reply:
correct

Reply:
confirmed offline this has not been added yet